website/docs/r/dashboard.html.markdown

```diff
@@ -160,6 +170,10 @@ The optional filter block supports the following arguments:
   * `event_types` - (Optional) A list of event types to enable filtering for.
   * `attributes` - (Optional) A list of attributes belonging to the specified event types to enable filtering for.
 
+## Additional Examples
+
+A
```

Comment:
Extra comment?

Reply:
Adding another example now, then this will be good to go I think.